features/openhab-addons-external/pom.xml

```diff
@@ -36,6 +36,7 @@
                   <header file="src/main/resources/addon-header.xml" filtering="no"/>
                   <fileset dir="${basedirRoot}/bundles">
                     <include name="*/src/main/resources/OH-INF/addon/addon.xml"/>
+                    <include name="*/src/main/resources/OH-INF/addon/addon-*.xml"/>
```

Comment:
Sorry, created a separate PR https://github.com/openhab/openhab-addons/pull/16044 for this. I hadn't seen you included it here.
You could simplify this to one line:
`<include name="*/src/main/resources/OH-INF/addon/addon*.xml"/>`

Reply:
rebased